### ISSUE TITLE
What is Rchain in Laymans Terms Episode 2 - Rholang

### ISSUE THREAD

**Issue by Cryptovideos** [rchain/bounties#954]
Benefit to RChain
A series of short animated productions explaining what RChain is about for the Layman. The series will help spread the word of what RChain is for the average person. Non-technical examples of what and how it works. 

In this installment we take a dive into Rholang and break it down.

- [x] Script dev & creation
- [x] Voice over production and editing
- [x] Animation and Video editing / special effects
- [x] Audio mixing and backing track

_Estimated Budget of Task:_ **$[1600]**
_Estimated Timeline Required to Complete the Task:_ **[1 week]**
_How will we measure completion?_ [https://www.youtube.com/watch?v=T7O7U2QtCP8&t=2s]

**Comment by Valentine-Mario** [comment]
Great work @Cryptovideos the first episode was epic and I trust this would be same.

**Comment by nonnykul** [comment]
nice **@cryptovideos** expecting to be amazed by this episode.

**Comment by Viraculous** [comment]
@Cryptovideos I see this milestone for this issue is completed and video uploaded on youtube. I am closing this issue.

**Comment by dckc** [comment]
I see @PatrickM727 budgeted $1600 for this; I just backed that vote with my own.
